README.md

```diff
@@ -362,12 +367,29 @@ Create a webhook or journal registration
 | Param | Type | Description |
 | --- | --- | --- |
 | consumerOrgId | <code>string</code> | Consumer Org Id from the console |
-| integrationId | <code>string</code> | integration Id from the console |
+| projectId | <code>string</code> | Project Id from the console |
+| workspaceId | <code>string</code> | Workspace Id from the console |
+| body | <code>object</code> | Json data contains details of the registration |
```

Comment:
How do people know what details to send in the payload? If we have an integration test, probably we can add a reference to that.
If this applies to most methods, then we can simpy add a generic reference to the integration tests. WDYT?

Reply:
let me see if we can add that also as a typedef

Reply:
added